code/datums/jobs/job/imperium.dm

```diff
@@ -147,7 +147,7 @@
 	H.equip_to_slot_or_del(new /obj/item/cell/lasgun/M43, SLOT_IN_BACKPACK)
 	H.equip_to_slot_or_del(new /obj/item/cell/lasgun/M43, SLOT_IN_BACKPACK)
 	H.equip_to_slot_or_del(new /obj/item/defibrillator, SLOT_IN_BACKPACK)
-	
```

Comment:
Unrelated whitespace changes

Reply:
I did attempt to reset, diff and stage it to no avail just right now.
Forgive me for being a slowpoke using shabby git GUIs on old potatoes in the attempt to save up time and effort. This is what I get for not doing --ignore whitespace on git bash.

Reply:
Are you using DM to code? Other editors dont' do this.

Reply:
Yes, I'm temporarily using a different potato and never bothered to switch to a better program (tried visualcode on the other one, but it was too memory-expensiveto work with), even though I have notepad++ installed and could use it.

Reply:
Sublime is all you need. It's better and more efficient than DM.

Reply:
Notepad++ is good as well. Poke me on discord if you need help solving these.